README.md

```diff
@@ -82,15 +84,15 @@ static/
 
 ## Publishing Post
 
-How to publish the blog on IPFS.io.
+How to publish the blog on `blog.ipfs.io`.
 
 ### Editing
 
 1. Make a change to a file
 2. Add and commit.
-3. `make build`
+3. `$ make build`
 4. `$ ipfs add -r build`
-  Only if you want a preview for other people (you can just use `make serve`). The path is `build`, in the website and the blog.
+  Only if you want a preview for other people. The path is `build`, in the website and the blog.
```

Comment:
Suggestion:
``` md 
4. `$ ipfs add -r build` [Optional]
  You can add the "build" directory to IPFS to get a hash containing your version of [the blog](https://ipfs.io/ipfs/Qmc5GSKzJ8qYfaVivfLxKHxTBfUXzYqBmgomKhzycnbMJz/). 
```
But I'm not actually sure if this is a good way to word it or demonstrate how to take the hash and turn it into a public preview of the render.  ┐('～`；)┌

Reply:
Hm.. this  may be scar tissue from time when blog and website were still a single thing.
I replaced the paragraph with simpler wording, without pointing at arbitrary CID: https://github.com/ipfs/blog/pull/199/commits/fed10b0093c2f815f9a90456faf70f2eafb8d04e